backend/pollpall_project/session/models.py

```diff
@@ -1,8 +1,41 @@
 from django.db import models
+from .models import Question
 
 class Session(models.Model):
     session_id = models.IntegerField(primary_key=True)
     session_label = models.CharField(max_length=255)
 
     def __str__(self):
-        return f"{self.session_id}: {self.session_label}"
\ No newline at end of file
+        return f"{self.session_id}: {self.session_label}"
+
+# TODO
+# By using ForeignKey, each question should be automatically connected to the poll
+# that fetches it. Each response is also connected to it's respective
+# question, so we shouldn't need a list of responses.
+class Poll(models.Model):
+    poll_id = models.IntegerField(primary_key=True) 
+    session = models.ForeignKey('Session', on_delete=models.CASCADE)
```

Comment:
nit: 
```suggestion
    session = models.ForeignKey(Session, on_delete=models.CASCADE)
```'

use the object Session and not a string